@xen-orchestra/fs/src/abstract.js

```diff
@@ -280,8 +280,8 @@ export default class RemoteHandlerAbstract {
     return this._read(typeof file === 'string' ? normalizePath(file) : file, buffer, position)
   }
 
-  async __readFile(file, { flags = 'r' } = {}) {
-    const data = await this._readFile(normalizePath(file), { flags })
+  async __readFile(file, { flag = 'r' } = {}) {
+    const data = await this._readFile(normalizePath(file), { flag })
```

Comment:
This does not work as `_readFile`'s default implementation uses `createReadStream` which uses `flags` and not `flag`!